x-pack/test_serverless/functional/test_suites/security/config.cloud_security_posture.ts

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { createTestConfig } from '../../config.base';
+
+export default createTestConfig({
+  serverlessProject: 'security',
+  junit: {
+    reportName: 'Serverless Security Cloud Security Functional Tests',
+  },
+  kbnServerArgs: [
+    `--xpack.fleet.packages.0.name=cloud_security_posture`,
+    `--xpack.fleet.packages.0.version=1.5.2`,
```

Comment:
I suspect that with these settings, the tests will not be able to run successfully against an actual MKI project.
If so, is there a way to change the test setup so it can run on MKI? Otherwise, the tests should be excluded from MKI runs by adding the suite tag `skipMKI` - but that would mean these tests are not part of our release test gates, so that should be an exception for the cases where it's really not possible to run a test on MKI but it's still considered valuable.

Reply:
Thanks, I will check if they run against an MKI. If not I think we can get away with adding an agent policy in the test suites, but this will make them slower for sure to add the policy before each suit. Maybe there are other ways to avoid package pre-instal, but we will need to investigate that.
btw what is the reason installing a package through config won't work on MKI? it's not configurable through Kibana server args?

Reply:
to not block the PR I will add `skipMKI` for now as I'm on PTO next week and create a follow-up to make the tests work on MKI

Reply:
> what is the reason installing a package through config won't work on MKI? it's not configurable through Kibana server args?

We won't do this in production for our customers. While there will be (or maybe already is?) an option to do project specific config overrides in MKI, this is not planned to be regularly done. So when we run functional tests against MKI, we want to be as close as possible to what our customers are getting and just spin up a project like a customer would do and point our tests to that instance. That means no extra server args, no test-only plugin loading, etc.

Reply:
thanks for the explanation! I added the tag to skip on MKI for now and will create a follow up to deal with it